python/plugins/processing/script/ScriptTemplate.py

```diff
@@ -53,11 +56,11 @@ def createInstance(self):
 
     def name(self):
         """
-        Returns the algorithm name, used for identifying the algorithm. This
-        string should be fixed for the algorithm, and must not be localised.
-        The name should be unique within each provider. Names should contain
-        lowercase alphanumeric characters only and no spaces or other
-        formatting characters.
+        Returns the algorithm name used for identifying the algorithm. This
+        string should be fixed for the algorithm and is not required to be
```

Comment:
In my opinion "localised" is a clearer phrasing then "in local language". But perhaps a compromise would be "and must not be translated"?